testsuite/openmodelica/dataReconciliation/TSP_Pipe3.mos

```diff
@@ -2247,15 +2247,24 @@ getErrorString();
 // -Passed
 //  Set_S has 58 equations and 58 variables
 //
+//
+// The selection of a new tearing variable failed.
 // record SimulationResult
 //     resultFile = "",
 //     simulationOptions = "startTime = 0.0, stopTime = 1.0, numberOfIntervals = 500, tolerance = 1e-06, method = 'dassl', fileNamePrefix = 'NewDataReconciliationSimpleTests.TSP_Pipe3', options = '', outputFormat = 'mat', variableFilter = '.*', cflags = '', simflags = '-reconcile -sx=./NewDataReconciliationSimpleTests/resources/DataReconciliationSimpleTests.TSP_Pipe3_Inputs.csv -eps=0.0023 -lv=LOG_JAC'",
 //     messages = "Simulation execution failed for model: NewDataReconciliationSimpleTests.TSP_Pipe3
-// assert            | debug   | Solving non-linear system 66 failed at time=0.
-// |                 | |       | For more information please use -lv LOG_NLS.
-// assert            | info    | simulation terminated by an assertion at initialization
+// LOG_SUCCESS       | info    | The initialization finished successfully without homotopy method.
+// LOG_SUCCESS       | info    | The simulation finished successfully.
 // stdout            | info    | DataReconciliation Starting!
 // stdout            | info    | NewDataReconciliationSimpleTests.TSP_Pipe3
+// stdout            | warning | The default linear solver fails, the fallback solver with total pivoting is started at time 1.000000. That might raise performance issues, for more information use -lv LOG_LS.
+// LOG_NLS           | error   | residualFunc76: Iteration variable xloc[0] is nan.
+// assert            | debug   | residualFunc76 failed at time=1.
+// |                 | |       | For more information please use -lv LOG_NLS.
+// assert            | debug   | Solving non-linear system 76 failed at time=1.
+// |                 | |       | For more information please use -lv LOG_NLS.
+// getBestJumpBuffer got simulationJumpBuffer=(nil)
+// Aborted (core dumped)
```

Comment:
👀

Reply:
I'm not sure what's supposed to happen. The model (before dataReconciliation) runs now, previously it failed at initialization. @arun3688 any idea what's going on here?

Reply:
I'll merge this for now to see the regressions. We can fix this in another PR.

Reply:
See #10109 for further discussion.